js/base/Exchange.js

```diff
@@ -1645,14 +1645,14 @@ module.exports = class Exchange {
         }
     }
 
-    networkCodeToId (networkCode) {
-        const networks = this.safeValue (this.options, 'networks', {});
-        return this.safeString (networks, networkCode, networkCode);
+    networkCodeToId (networkCode, currencyCode = undefined) {
```

Comment:
@ttodua why add `currencyCode` here if it is not used?

Reply:
> @ttodua why add `currencyCode` here if it is not used?

Maybe I haven't told that I had updated initial top-post which explains the reason, but to again mention, for example, initially, for huobi I needed to make an override of this method in derived class, so the signature needs to match. and for huobi, this signature was needed, where `currencyCode` is included. For huobi, something like this logic was needed:
```
networkCodeToId (networkCode, currencyCode = undefined) {
    ....
    return currencyCode.toLowerCase() + networkCode;
}
```

so, if we don't change signature, then we can't make an override in derived class. also, to mention again, this will not be breaking change, and also I think that there is no much room left for its signature's changes in future, as these `networkCodeToId` methods doesn't have any need of other arguments, only `networkCode` and `currencyCode` is maximum that might be ever needed for this method signature.

Reply:
@ttodua if this is a huobi-specific need, I think we should isolate it here, you can always create a `customNetworkCodeToId`  there

Reply:
> @ttodua if this is a huobi-specific need, I think we should isolate it here, you can always create a `customNetworkCodeToId` there

yes, we could use that, but there are other methods which has reference/use `networkCodeToId` (i.e. `selectNetworkIdFromAvailableNetworks`), so that's why we needed overriden `networkCodeToId` method, and not `custom...`.   also, adding `currencyCode` in signature  doesn't hurt anything, isn't that right  (we have added these methods to basecode recently)?

Reply:
@ttodua I'm afraid that we might be confusing some devs by thinking that the `currencyCode` does something useful by default and if Huobi is the only one requiring this parameter we should not design a generic widely-available method around it, some other exchange might require the extra X param and we're not going to add it to the signature right?

Reply:
@carlosmiei of course, I understand what you say (and have been aware of that before doing this update), but I don't see any issue with this, as long as it has default value, and it doesn't hurt being there (even if anyone passed there currencyCode, it wouldnt hurt) and moreover, the default value `currencyCode = undefined` expresses that it's not required/needed. 
It might be for huobi-only at this moment, but I am sure that none of other exchange will need any other `x` parameter additionally, at maximum the second argument that could have been ever required for this functionality, could have been 
 `currencyCode`, so function could understand (whenever needed) for which `currency` it sets `networkCode`.
otherwise, when exchange requires the `currencyCode` too to make internal recongition & parsing networks (like huobi) then we have to add multiple `customMethod` implementations inside such derived class, while we could easily override `networkCodeToId` which would pass `currencyCode` to overrided implementation. just I think implementing it has more benefit, than downsides, but as you @lead decide, let be so. lmk the decision and I wouldn't make further hesitation✌️

Reply:
@ttodua so can you add a small comment inside the base implementation explaining that `currencyCode` is not used but it's there to simplify some overrides inside derived exchanges pls?

Reply:
> @ttodua so can you add a small comment inside the base implementation explaining that `currencyCode` is not used but it's there to simplify some overrides inside derived exchanges pls?

sure, done!